models/hds.go

```diff
@@ -320,7 +320,7 @@ var hqmf_qrda_oids = []byte(`[
         "attribute": "",
         "exclude_null_flavor": false,
         "extra_content": "xsi:type=\"CD\" sdtc:valueSet=\"{{.MapDataCriteria.ValueSetOid}}\"",
-        "required_code_sets": ["2.16.840.1.113883.3.88.12.3221.7.4"],
```

Comment:
@sylinmino should review this section at least; I think the change from preferred to required was done by him.